library.py

```diff
@@ -11,10 +11,13 @@
 import util
 
 config = configparser.ConfigParser()
-config.read("config.ini")
+config_path = os.environ.get("DOCUGEN_CONFIG_PATH") or "./config.ini"
```

Comment:
In the Action, we're not in the same directory as this script when we run it, so this lets us load the config from the repo manually